docs/serving/autoscaling/autoscale-go/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Autoscale Sample App - Go"
-linkTitle: "Autoscaling - Go"
-weight: 1
+linkTitle: "Autoscale Sample App - Go"
+weight: 100
 type: "docs"
```

Comment:
You may want to add an `aliases` here to allow the old links to also work.

https://gohugo.io/content-management/urls/#aliases

Reply:
Hopefully I've done these right now :slightly_smiling_face: